apps/i18n/netsim/en_ploc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "yes": "Yes"
```

Comment:
We may want to add a task to audit these app localization files, and move very common localizations into the common file.  `"yes": "Yes"` shows up in the individual localization files for every app.  I wonder how many other duplicates we've got.

![screenshot from 2015-01-23 12 28 27](https://cloud.githubusercontent.com/assets/1615761/5882126/8a0e4250-a2fb-11e4-81cc-0beac8f0615e.png)